testcases/kernel/syscalls/setgroups/setgroups03.c

```diff
@@ -191,6 +206,12 @@ int setup1(void)
 {
 	struct passwd *user_info;	/* struct. to hold test user info */
 
+	// Nobody user do not have permissions to open /etc/passwd file 
```

Comment:
224 is fixed. This isn't needed.

Reply:
Yes https://github.com/lsds/sgx-lkl/pull/723 fixed this. This needs to be removed. Not needed anymore.

Reply:
Done